### PR TITLE
Article/fed talk s01e01 getting started

### PR DESCRIPTION
**Pull request by qt-pixels**

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -11,25 +11,27 @@ <h1 class="post-title p-name" itemprop="name headline">{{ page.title | escape }}
...
+    {%- for tag in page.tags -%}
+    <span style="margin-right: 3px">
+      <a href="/tags/#{{ tag }}">{{tag}}</a>
+    </span>
+    {%- endfor -%}
+    {%- endif -%}
   </header>
 
-  <div class="post-content e-content" itemprop="articleBody">
+  <div class="post-content e-content {{page.font}}" itemprop="articleBody">
```
Comment (qt-pixels) [inline _layouts/post.html:28]: Here is where I plug in the new Font styles from the Articles header variables.

File: assets/style.css [hunk 2]
```diff
@@ -37,4 +36,39 @@ a, a:visited {
 .footnote {
     font-size: "-2";
     color: #666;
-}
\ No newline at end of file
+}
+
+.post-content.serif {
```
Comment (qt-pixels) [inline assets/style.css:41]: New Font styles applied here to the bottom of the page.

File: _posts/2021-03-23-fed-talk-s01e01-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+font: serif
```
Comment (qt-pixels) [inline _posts/2021-03-23-fed-talk-s01e01-getting-started.md:3]: New Font variable

File: _posts/2021-03-23-fed-talk-s01e01-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+tags: front-end-development FED-Talk react material-ui design theme
+author: Quintin Maseyk
+image: /img/fed-talk/cover-ep1-740.png
+---
+
+![Front-End Development Talk: Episode 1 - Getting Started](/img/fed-talk/cover-ep1-740.png)
+
+I’ve been meaning to write a Front-End Developer (FED) series for quite some time now. Having worked in many industries, building many Web Apps on different stacks and Design Systems, I’m here to share my learnings and hopefully you will get something out of it.
+
+These days it's very easy for new developers to get up and running using tooling such as `create-react-app`. Tooling such as this are great as it speeds up repetitive tasks and automates all of a React's initial constructs, enabling developers to be able to build, test and deploy their applications. But, great tooling can also restrict one's knowledge of understanding what actually happens during and after its compilation.
```
Comment (Hamo69) [inline _posts/2021-03-23-fed-talk-s01e01-getting-started.md:15]: Tooling such as this are great
should be
Tooling such as this is great

**Review by Hamo69 — APPROVED** [review]

**Pull request merged**